tests/WorkOS/DirectorySyncTest.php

```diff
@@ -361,7 +361,7 @@ private function usersResponseFixture()
                     ],
                     "raw_attributes" => [
                         "schemas" => ["urn:scim:schemas:core:1.0"],
-                        "name"=> [
+                        "name" => [
```

Comment:
Let's revert the changes to this file since they are unrelated to our other changes.